src/Components/CatProfile.tsx

```diff
@@ -125,14 +125,10 @@ function CatProfile() {
 					value={catDescription}
 					onChange={handleProfileEditBy}
 				/>
-				<Button onClick={handleSavedChanges}>Save</Button>
+				<Button onClick={handleSavedChanges}>Submit</Button>
```

Comment:
I didn't explain this well. It's not the text that needed to be `Submit`. You make a button as the default button like this:

`<button type="submit">Save</button>`